docs/scaleway.md

```diff
@@ -0,0 +1,41 @@
+# Example.com
+
+## Configuration
+
+If something is unclear in the documentation below, please refer to the [scaleway API documentation](https://www.scaleway.com/en/developers/api/domains-and-dns/#path-records-update-records-within-a-dns-zone).
+
+### Example
+
+```json
+{
+    "settings": [
+        {
+            "provider": "scaleway",
+            "domain": "munchkin-academia.eu",
+            "secret_key": "<SECRET_KEY>",
+            "ip_version": "ipv4",
+            "ipv6_suffix": "",
+            "field_name": "www",
+            "ttl": 450
+        
+        }
+    ]
+}
+```
+
+### Compulsory parameters
+
+- `"domain"` is the domain to update. It can be `example.com` (root domain), `sub.example.com` (subdomain of `example.com`) or `*.example.com` for the wildcard. This fields corresponds to the `dns-zone` in the scaleway API documentation.
```

Comment:
```suggestion
- `"domain"` is the domain to update. It can be `example.com` (root domain), `sub.example.com` (subdomain of `example.com`) or `*.example.com` for the wildcard. This field corresponds to the `dns-zone` in the scaleway API documentation.
```